recipes/tests.py

```diff
@@ -348,3 +348,55 @@ def test_non_user_unlike_recipe(self):
             reverse('like-recipe', kwargs={'pk': recipe.id})
         )
         self.assertEqual(response.status_code, status.HTTP_401_UNAUTHORIZED)
+
+
+class CreateUpdateRatesTestCase(BaseTestCase):
+    def test_can_user_create_rate_to_recipe(self):
+        user, client = self.create_user()
+        recipe = self.create_recipe(user=user)
+        score = 5
+        response = client.post(
+            reverse('rate-recipe', kwargs={'pk': recipe.id}),
+            {'score': score}
+        )
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(Rate.objects.first().score, score)
+        self.assertEqual(Rate.objects.first().user, user)
```

Comment:
You can combine these two assertions -> `assertTrue(Rate.objects.filter(user=user, score=score).exists())`